app/models/accounting/quickbooks/fetcher.rb

```diff
@@ -0,0 +1,53 @@
+module Accounting
+  module Quickbooks
+    class Fetcher
+      def initialize(relation)
+        @relation = relation
+      end
+
+      def query
+        util = ::Quickbooks::Util::QueryBuilder.new
+
+        types.map do |type|
+          q = "select * from #{type} where #{util.clause('Id', 'in', ids)}"
+          service(type).query(q).entries
+        end.flatten
+      end
+
+      def populate(qb_objects)
+        @relation.each do |transaction|
+          qb = qb_objects.find { |qbo| qbo.id.to_s == transaction.qb_transaction_id }
+          transaction.qb_object = qb
+        end
+        @relation
+      end
```

Comment:
Can the above two methods please be private? I was confused because I couldn't understand why all three would need to be used by outside objects, but it seems only fetch does, right?